### DISCUSSION TITLE
Add label to EdgeLine

### DISCUSSION THREAD

**Discussion by dgalv** [sgratzl/chartjs-chart-graph#28] [Ideas]
I use your plugin for a physical approch. For me a point is an equipment and an edgeline is a link between two equipments.
I would like display additional data related to each link in my graph.
Is possible to add label to a line ?

I use the Force Directed Graph with the datalabels plugin

Thx for your work

Regards,

**Comment by sgratzl** [comment]
so far this plugin doesn't support any labeling. Node labels are supported via chart-plugin-datalabels plugin. Have you looked at other libraries such as https://js.cytoscape.org/

**Comment by darrenhull** [comment]
We too need labels on the edges and might need to move to a different library if this one doesnt support them.
